src/routes/AuthLayoutRouter/AuthLayoutRouter.js

```diff
@@ -176,27 +175,16 @@ export default function AuthLayoutRouter (props) {
         <RedirectRoute exact path='/:context(groups)/:groupSlug' to={`/groups/${currentGroupSlug}/explore`} />
       )}
 
-      <Route path='/:context(groups)/:groupSlug' render={routeProps => <GroupWelcomeModal {...routeProps} />} />
+      {!hyloAppLayout && (
```

Comment:
@tibetsprague here is where the `hyloApp` (previously `mobileSettings`) the layout flags begin to diverge from the alias we're currently using of `withoutNav`. It's your call on whether you want to use a custom layout flag (`cafeLayout`?) or simply start using the general `withoutNav` as the actual querystring flag (e.g. `?layoutFlags[]=withoutNav|cafeLayout`). I think the more custom flag is maybe best. Regardless, whatever you do there you'll want to keep `hyloAppLayout` added to the conditionals where they currently are also. Hope that makes sense. Reach-out if you want clarification.

Reply:
The only thing you need to start using an additional layout flag is add it's name to the whitelist in `LayoutFlagsContext` and just add it to the querystring param `layoutFlags`, noting that the word `Layout` is appended by the context so the flag itself shouldn't have that in the name.